### DISCUSSION TITLE
How to show skipped content in stack?

### DISCUSSION THREAD

**Discussion by rew1nter** [pwndbg/pwndbg#2074] [Q&A] [answered]
Why's this skipping these 2 blocks and how can I show this?
![image](https://github.com/pwndbg/pwndbg/assets/64508791/d9d4e5ff-08ff-4113-bfec-bf616acfb5f9)

**Comment by RocketMaDev** [comment]
Is there a need to show that? Skipping blocks aims to fold duplicate contents, in your case, esp is 'AAAA', esp+4 is 'AAAA' and esp+8 is also 'AAAA', so 2 blocks is skipped. Skipped blocks means contents in these blocks are the same as the block before skipped ones.

**Comment by disconnect3d** [comment, marked as answer]
Hi, sorry for the late reply!

Pwndbg can be configured with debugger parameters. GDB has a lot of its own parameters like `arch` or `language` but we define our own Pwndbg ones too.

The two interesting parameters that we have here are:
```
telescope-skip-repeating-val               True       whether to skip repeating values of the telescope command
telescope-skip-repeating-val-minimum       3          minimum amount of repeated values before skipping lines
```

You can do `set telescope-skip-repeating-val off` to disable the skipping of repeated values.

You can also see all Pwndbg parameters with the `config`, `theme` and `heap_config` commands. They can also be filtered via `config <text>` so `config skip` will show the two parameters listed above.

Hope this helps!

Btw for future, I'd recommend joining our Discord - we answer there much faster then here!
